sale_order_type/models/sale_order_type.py

```diff
@@ -52,10 +52,11 @@ class SaleOrderTypology(models.Model):
         ondelete="restrict",
         check_company=True,
     )
-    analytic_account_id = fields.Many2one(
-        comodel_name="account.analytic.account",
-        string="Analytic account",
-        check_company=True,
+    project_id = fields.Many2one(
+        comodel_name="project.project",
+        domain=[("allow_billable", "=", True)],
+        string="Project",
+        help="Select to define the analytics account (new approach in v18)",
```

Comment:
We don't need to add a changelog here.
```suggestion
        help="Select to define the analytics account",
```

Maybe add `check_company=True` here.

Reply:
Any type of company check between order type and project (like `check_company` on the old analytic account field)  still seems mising.